ftrackWidget.js

```diff
@@ -1,253 +1,246 @@
-'use strict';
-
-(function (root, factory) {
-    if (typeof define === 'function' && define.amd) {
-        // AMD. Register as an anonymous module.
-        define([], factory);
-    } else if (typeof module === 'object' && module.exports) {
-        // Node. Does not work with strict CommonJS, but
-        // only CommonJS-like environments that support module.exports,
-        // like Node.
-        module.exports = factory();
-    } else {
-        // Browser globals (root is window)
-        root.ftrackWidget = factory();
+/**
```

Comment:
Consider if we should move this file to e.g. `src/ftrackWidget.js`, and maybe add a `src/index.js` which re-exports everything in it.

Reply:
done